app/src/main/java/eu/kanade/tachiyomi/data/backup/restore/BackupRestorer.kt

```diff
@@ -157,10 +166,33 @@ class BackupRestorer(
         )
     }
 
+    private fun CoroutineScope.restoreExtensionRepos(
+        backupExtensionRepo: List<BackupExtensionRepos>
+    ) = launch {
+        backupExtensionRepo
+            .forEach {
+                ensureActive()
+
+                try {
+                    extensionRepoRestorer(it)
+                } catch (e: Exception) {
+                    errors.add(Date() to "Error Adding Repo: ${it.name} : ${e.message}")
+                }
+
+                restoreProgress += 1
```

Comment:
+1 progress for each repo while only +1 to restore amount for the settings being enabled?
https://github.com/mihonapp/mihon/blob/b6f5c6cde77222f9d1c689692ebc0176d0988d4e/app/src/main/java/eu/kanade/tachiyomi/data/backup/restore/BackupRestorer.kt#L82-L84

Reply:
My bad, forgot to switch it after this commit 6b1f79c46edfbb8067bd624462bfdf802ce8fcd5